arm-recoveryservices/2015-11-10/swagger/replicationusages.json

```diff
@@ -0,0 +1,339 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-11-10",
+    "title": "RecoveryServicesClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": false
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "description": "Returns the list of available operations.",
+        "operationId": "Operations_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
```

Comment:
nit: maintain consistency, either place ```SubscriptionId``` and ```ApiVersion``` together at the beginning of the parameters section or at the end